src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -143,7 +132,7 @@ private static void updateAnimIndex()
   public static BufferedImage buildCoPowerBar(Commander co, double[] abilityPoints, double currentPower, boolean leftSide)
   {
     final double pixelsPerPowerUnit = 3.0;
-    int slowAnimIndex = (animIndex/32) % 2;
+    int slowAnimIndex = (getAnimIndex()/32) % 2;
```

Comment:
`getAnimIndex()` is called multiple times in this function - maybe just call it once and store it instead of recalculating each time?